tests/test_models.py

```diff
@@ -257,7 +283,21 @@ def test_ps_torch_output(self, n_scales, n_orientations,
             output.squeeze(), saved.squeeze(), rtol=1e-5, atol=1e-5
         )
 
-    def test_ps_synthesis(self, portilla_simoncelli_synthesize):
+    def test_ps_synthesis(self, portilla_simoncelli_synthesize,
+                          run_test=True):
```

Comment:
Oh I see. This is so the utilities function can use exactly the same code to generate the metamer.

Is it clearer if this  argument is return_metamer instead (False by default)?

Reply:
Yup. There are two components that I need to reuse this: I need the metamer object to be returned and the tests to not be run, so I felt that `run_test` made sense. Plus, if we need to do something similar for other situations, they might not necessarily involve a metamer object, but they would need to avoid running the tests

Reply:
Somehow I don't think that I actually read the parameter definitions when I got confused about this 🤦‍♀️ .  That seems reasonable to me.